stuff/lib/logs.py

```diff
@@ -70,13 +71,13 @@ def init(program: str, args: argparse.Namespace) -> None:
             log_handler: logging.Handler = logging.FileHandler(args.logfile)
         else:
             log_handler = logging.StreamHandler()
-        formatter = _CustomJsonFormatter()
+        formatter: _CustomJsonFormatter = _CustomJsonFormatter()
```

Comment:
this is not complaining about the lack of a type annotation here, but rather at the declaration of the class. we can ignore it for now:

```suggestion
        formatter = _CustomJsonFormatter(). # type: ignore
```